ansible/library/conn_graph_facts.py

```diff
@@ -291,14 +311,14 @@ def main():
             LAB_GRAPHFILE_PATH = m_args['filepath']
 
         if m_args['filename']:
-            filename = m_args['filename']
+            filename = os.path.join(LAB_GRAPHFILE_PATH, m_args['filename'])
```

Comment:
Why do we have a requirement that the file has to be in this path?

Reply:
Without the path, ansible will try to lookup the graph files under the CWD. We need to tell ansible where to lookup the graph files. Generally, all the graph files are stored under `LAB_GRAPHFILE_PATH`. This global variable has default value `sonic-mgmt/ansible/files`. Its value can be overridden by passing in argument `filepath` while calling the `conn_graph_facts` module. No matter use default value or use customized value, `LAB_GRAPHFILE_PATH` is always for location of the graph files. So, we should always try to lookup graph files under `LAB_GRAPHFILE_PATH `